### PR TITLE
Update E2E packages

### PR DESCRIPTION
**Pull request by rylew1**
## Ticket

Resolves https://github.com/navapbc/template-infra/issues/801

## Changes

- Update npm packages and bump Dockerfile playwright image version

## Preview environment for app
- Service endpoint: http://p-94-app-dev-1070269506.us-east-1.elb.amazonaws.com
- Deployed commit: 7f5866407427eb607140915a053da910a085d337

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -112,7 +128,9 @@ e2e-test: e2e-build
 		-e CI=$(CI) \
 		-v $(PWD)/e2e/playwright-report:/e2e/playwright-report \
 		-v $(PWD)/e2e/blob-report:/e2e/blob-report \
-		playwright-e2e
+		playwright-e2e | grep -v "To open last HTML report run" | grep -v "npx playwright show-report"
+	@echo "📄 HTML report generated!"
+	@echo "🔍 View it with: make e2e-show-report"
```
Comment (rylew1) [inline Makefile:133]: remove the default playwright command it shows and insert our make command

Comment (rylew1) [reply Makefile:133]: What about not removing the the playwright command but just echo'ing the make command ?

Will move just the package updates over to `template-infra`

**Comment by rylew1** [comment]
Moved to `template-infra` with just package updates https://github.com/navapbc/template-infra/pull/805

File: Makefile (hunk 1 above)
Comment (rylew1) [reply Makefile]: @lorenyu see above comment if it makes sense to do

Comment (lorenyu) [reply Makefile]: > What about not removing the the playwright command but just echo'ing the make command ?

Sure that sounds good

**Pull request closed without merging**